Readability.js

```diff
@@ -2029,6 +2030,10 @@ Readability.prototype = {
         return false;
       }
 
+      if (this._hasAncestorTag(node, "code", -2)) {
```

Comment:
Why -2 ? In the other change you didn't pass a `maxDepth` argument at all. It also looks like passing `0` or `-2` is functionally equivalent, so this is a bit confusing. Can you clarify, or maybe change to not pass this if it doesn't make a difference? Thanks!